src/Propel/Common/Config/Loader/IniFileLoader.php

```diff
@@ -28,7 +28,7 @@ class IniFileLoader extends FileLoader
      *
      * @var string
      */
-    private $nestSeparator = '.';
+    private string $nestSeparator = '.';
```

Comment:
We didnt add any such types on properties yet. This would be the first.

Reply:
I am not against it, just outlining that it can have risks.
As a first step the public API (methods that are non private) should be the focus.

Reply:
I'll remove it if you want, but I don't see the issue in a gradual approach to more strict typing.

Reply:
I think the ones that do not default are the problem, as they need to be written first, before they can be accessed.
So I guess this is fine here.